docker-compose.yml

```diff
@@ -169,6 +181,8 @@ services:
       - infrahub-server
     environment:
       <<: *infrahub_config
+      INFRAHUB_API_TOKEN: *agent_token
```

Comment:
If we use the agent token here it won't be possible to run the invoke commands to load the edge data script as it's being executed from the git-agent and with this change the git-agent container isn't allowed to create accounts which the demo data script is doing.